curriculum/test-challenges.js

```diff
@@ -189,18 +196,24 @@ function createTest({
     });
 }
 
-Observable.from(getChallenges())
-  .do(({ challenges }) => {
-    challenges.forEach(challenge => {
-      const result = validateChallenge(challenge);
-      if (result.error) {
-        console.log(result.value);
-        throw new Error(result.error);
-      }
-    });
+Observable.fromPromise(getChallengesForLang('english'))
```

Comment:
```suggestion
Observable.fromPromise(getChallengesForLang(LOCALE || 'english'))
```

We can derive the locale variable from the `.env`, let's just make it default.

Reply:
Do we need to test all languages on CI/CD?

Reply:
Yes, the languages are essentially forks of the english version, and as a matter of fact will always be behind the english versions. Meaning broken tests will have to be fixed in every language version again.

Also, the CD will be responsible for deploying the language versions of the applications to their own "hosting" locations. 

We would build these in parallel as per the language (actually its more complicated, I am over simplifying the workflow a bit).

Reply:
fixed